### DISCUSSION TITLE
Ampersand issues when inserting data to database

### DISCUSSION THREAD

**Discussion by daniel-lagrava** [dtcenter/METplus#2048] [Plot Generation] [answered]
Hello, 

I have been working with thresholds of the form `>=31.25&&<56.25` for a dichotomous variables. MET handles all the statistical part well. However, when I try to insert that data to METdatadb, it will cause a disconnection from the DB server and result in an error inserting the data. I have converted such strings to `>=31.25AND<56.25` and then the insertion works well. Are you aware of another way to do this or do I need to keep pre-processing the files before insertion. Note: I am using MET coordinated release v4.1 and have not yet tested this use case with MET coordinated release v5.0, but I will soon update our systems. (cc @lindsayrblank)

Thanks a lot,
Daniel

**Comment by j-opatz** [comment]
Hi Daniel,

And thank you for your question. I'm going to tag @venitahagerty  on this issue for confirmation on this potential read error for the METdataio application and hopefully she can provide more insight on this.

**Reply by venitahagerty** [reply]
Hi Daniel,

I changed a small data file to have a CTC line with a forecast threshold of >=31.25&&<56.25 and it loaded with METdbload. Could you please attach a small data file that has this threshold value? Also, how are you loading these files, and do you get any error messages? Thank you

**Comment by daniel-lagrava** [comment, marked as answer]
Hello Venita,

Thanks a lot for the answer. I tested again with a smaller file and it does indeed work! It seems like a had a DB disconnection several times when trying to insert the CTC file not related to the actual content, even though it worked for the SL1L2 files. I thank you for your time and please consider this discussion closed, it is my bad.

Cheers!
